sentry/src/main/java/io/sentry/NoOpSpan.java

```diff
@@ -0,0 +1,59 @@
+package io.sentry;
+
+import io.sentry.protocol.SentryId;
+import org.jetbrains.annotations.NotNull;
+import org.jetbrains.annotations.Nullable;
+
+public final class NoOpSpan implements ISpan {
+
+  private static final NoOpSpan instance = new NoOpSpan();
+
+  private NoOpSpan() {}
+
+  public static NoOpSpan getInstance() {
+    return instance;
+  }
+
+  @Override
+  public @NotNull ISpan startChild() {
+    return new NoOpSpan();
+  }
+
+  @Override
+  public @NotNull ISpan startChild(@Nullable String operation, @Nullable String description) {
```

Comment:
should `operation` but NonNullable and set to `noop` instead?

Reply:
Has it been documented somewhere? "op" is not a required field (https://develop.sentry.dev/sdk/event-payloads/transaction/)

Reply:
Apparently it's required:

![image](https://user-images.githubusercontent.com/1935960/104929583-c0deb580-59ac-11eb-9980-61cd4c9e0b5d.png)
https://discord.com/channels/621778831602221064/740915087220408350/800720296432500746

Reply:
![image](https://user-images.githubusercontent.com/1357927/104936126-66455980-59ac-11eb-8721-f4ed82a02fd5.png)
@marandaneto please clarify this.

Reply:
cc @rhcarvalho

Reply:
See https://github.com/getsentry/sentry-java/issues/1170

Reply:
👍